nx/lib/nx.ex

```diff
@@ -1281,9 +1281,9 @@ defmodule Nx do
         ]
       >
 
-  ## Vectorized tensors
+  ## Vectors
```

Comment:
I prefer the previous titled. We already have `~V` each means one dimensional tensor. So calling it "Vectors" here is confusing. :(

Reply:
Let's keep it as vectorized tensors because vectors is really ambiguous.